src/chigame/api/serializers.py

```diff
@@ -30,3 +30,27 @@ class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
         fields = ("id", "name", "email")
+
+
+class MessageSerializer(serializers.ModelSerializer):
```

Comment:
I am getting this error when I try to post
<img width="739" alt="image" src="https://github.com/uchicago-cs/chigame/assets/102563648/a3b445e4-c6b3-42b6-9081-1e71bfaff5d0">
I'm also seeing a 405 error
<img width="342" alt="image" src="https://github.com/uchicago-cs/chigame/assets/102563648/1f7a3c1a-3c6d-4906-b043-b0362ab2d892">

Reply:
For the DNE error, it seems like the POST request is sending a tournament ID that does not yet exist in the database. Please make sure that a valid tournament ID is provided and a chat is created for that tournament in the database before sending the request. We haven't yet included an error handling for this, as in our later PRs, we will auto-create a chat for every tournament once a tournament is created, and we will only send a request to this endpoint with a valid tournament ID from the front end. If you feel like we should have some error handling for this entry point, I'll definitely go ahead and add one.

For the 405 error, the error message you provided says "Method GET is not allowed"; for that endpoint, we only allow a POST method. Let's switch to the POST method and then see if the 405 error persists.

### Sample Post request body

__make sure you have a tournament with pk 1 and have created a chat for it__

{
    "sender": "admin@example.com",
    "content": "This is a test script 1",
    "update_on": null,
    "tournament": "1"
}

Reply:
In later reviews, if we wish to test the API, we can also use the test_api.py file to test if the API is behaving logically:

```
python3 manage.py test chigame.api.tests.test_api.ChatTests
```